canonical/juju.is
# feat: update operator day

## Done

Update operator day pages

## QA

- Visit the demo
- Check the operator day pages match the copy doc: https://docs.google.com/document/d/1ZFRGkZ4vvvMVOgq4MIsBuebHsA7Y0rTyiZ76qjQpOAg/edit#heading=h.se0ejigxdrtm


## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-8227
